docs/docs/5min-tutorial.mdx

```diff
@@ -181,3 +181,42 @@ order to avoid conflicts with other tutorials from this guide:
 $ docker-compose -f quickstart.yml kill
 $ docker-compose -f quickstart.yml rm -f -v
 ```
+
+### Quickstart Configuration
+
+In this tutorial we use a simplified configuration.  
+You can find it in `contrib/quickstart/5-min/hydra.yml`.  
+The configuration gets loaded in docker-compose as specified in the
+`quickstart.yml`.
+
+```yml
```

Comment:
Can you link to the file? It is also possible to import files from GitHub directly, check the kratos docs (specifically I think the login guide) for how to do that

Reply:
Ok! 
Don't know if I can do it today, but tomorrow latest.